Fix/nft metadata and label issues

jorbuedo opened the pull request.

jorbuedo committed:
fix: resolve NFT metadata errors and improve component styling

- Fix TypeError in NftMetadata component by adding null checks
- Simplify NftMetadata to return null instead of hardcoded messages
- Fix LabelConnected component to use proper themed atoms
- Fix usePortfolioImage hook useRef initialization error
- Remove inline styles in favor of atomic design system

jorbuedo committed:
feat: integrate Claim feature into navigation

- Add ShowSuccessScreen to TxHistoryNavigator
- Connect claim-show-success route to proper navigation
- Ensure Claim feature is fully accessible through navigation system

jorbuedo committed:
fix claim modal

jorbuedo committed:
fix clami modal

stackchain commented on apps/mobile/src/features/Portfolio/ui/MediaDetailsScreen/MediaDetailsScreen.tsx:
```diff
@@ -162,11 +164,16 @@ const SelectableMedia = ({info}: {info: Portfolio.Token.Info}) => {
+        flexDirection: 'row',
+        flexGrow: 1,
+        backgroundColor: p.gray_100,
+      }}
```
```suggestion
      style={[
        a.flex,
        a.flex_row,
        a.flex_grow,
        {backgroundColor: p.gray_100},
      ]}
```

stackchain committed:
Update apps/mobile/src/features/Portfolio/ui/MediaDetailsScreen/MediaDetailsScreen.tsx

Signed-off-by: Juliano Lazzarotto <30806844+stackchain@users.noreply.github.com>

stackchain commented on apps/mobile/src/features/Portfolio/ui/MediaDetailsScreen/MediaDetailsScreen.tsx:
```diff
@@ -162,11 +164,16 @@ const SelectableMedia = ({info}: {info: Portfolio.Token.Info}) => {
       <MediaPreview
         info={info}
-        style={[{flexGrow: 1, backgroundColor: p.gray_100}]}
+        style={{}}
```
isnt optional?

stackchain commented on apps/mobile/src/features/Portfolio/ui/MediaDetailsScreen/MediaDetailsScreen.tsx:
```diff
@@ -375,7 +388,7 @@ const NftMetadata = ({discovery}: {discovery: Portfolio.Token.Discovery}) => {
     <View>
       <Copiable
-        title={strings.portfolio.nftDetail.copyMetadata}
+        title={strings.txReview.tokenDetails.jsonTab.metadata}
```
this should be portfolio

jorbuedo replied:
So many repeated ones...

jorbuedo committed:
fix: resolve MediaPreview component styling and type issues

- Replace inline styles with atomic design tokens (a.relative, a.overflow_hidden, a.absolute, a.inset_0, a.z_10)
- Fix Image source prop type handling for expo-image compatibility
- Fix prettier formatting issues
- Improve code consistency with atomic design system

jorbuedo committed:
fixes

jorbuedo committed:
Merge branch 'develop' into fix/nft-metadata-and-label-issues
